### ISSUE TITLE
feature - defining failure criteria

### ISSUE THREAD

**Issue by tooptoop4** [YotpoLtd/metorikku#169]
would be nice to have a step where you could define failure/continue rules:

step1:
sql: select count(*) count from df where (length(run_date) > 0 and to_timestamp(run_date,'yyyy-MM-dd HH:mm:ss') is null )
failCriteria: count > 0
step2:
sql: whatever next step was.....

metorikku would exit out with failure (returncode 1) after step1 if this sql returned count > 0, otherwise if count is 0 then metorikku can proceed to step2

**Comment by lyogev** [comment]
Cool idea I would actually just create a new type of step that continues/exits based on query outcome.

**Comment by tooptoop4** [comment]
workaround:
add this where to step2 sql --> where 1 = (select 1 from step1 where count > 0 union all select 1 from step1 where count > 0 union all select 1 from step1 where count = 0 ) so it will force error of multi rows returned if count > 0 only
